src/realtime_metrics/main.py

```diff
@@ -148,6 +180,52 @@ def eta_accuracy(stop_time_updates: list[tuple[TripUpdate, StopTimeUpdate]]):
     return mean_accuracy
 
 
+def availability_acceptable_stop_time_updates(stop_time_updates: list[tuple[TripUpdate, StopTimeUpdate]], 
+                                              time_frame_start: int, 
+                                              time_frame_end: int):
+    """
+    computes the availability of acceptable stop time updates metrics for the given stop time updates in the given time frame.
```

Comment:
Maybe we can add a sentence or two to explain what the calculation does and link to the document explaining it